### PR TITLE
[HOLD] [v2] Remove duplicated types

### PR DESCRIPTION
**Pull request by alloy**
Closes https://artsyproduct.atlassian.net/browse/PLATFORM-1582

- [x] `ArtworkContext*` (replace w/ actual types i.e. sale)
- [x] `ArtworkFilter*` (review for possible replacement w/ Tag)
- [x] `Highlighted*`
- [x] `HomePageModuleContext*`
- [x] `Category` (Remove in V2 in favor of `PartnerCategory`)
- [ ] ~~`CategoryType` (Remove in V2 in favor of `PartnerCategoryType`)~~ Deferring this till we get rid of transforms, see JIRA ticket for details.
- [x] `ItemType`
- [ ] ~~Make things work again when enabling removal of deprecated types.~~ Deferring this till we get rid of transforms, see JIRA ticket for details.

### CODE REVIEW

File: _schemaV2.graphql [hunk 1]
```diff
@@ -1678,28 +1679,19 @@ type ArtworkContextFair {
 type ArtworkContextPartnerShow implements Node {
   # A globally unique ID.
   id: ID!
-    @deprecated(
```
Comment (zephraph) [inline _schemaV2.graphql]: Why were these removed?

File: src/schema/home/context.ts [hunk 2]
```diff
@@ -180,9 +181,78 @@ const Context: GraphQLFieldConfig<
...
     return moduleContext[key](context, params)
   },
 }
 
-export default Context
+const HomePageArtworkOfRelatedToFollowedArtistModuleType = new GraphQLObjectType<
+  any,
+  ResolverContext
+>({
+  name: "HomePageArtworkOfRelatedToFollowedArtistModule",
```
Comment (alloy) [inline src/schema/home/context.ts]: @zephraph Perhaps this should just be `Artist` and we can add to the description of the `context` field that `Artist` represents a followed artist?

File: src/schema/home/context.ts [hunk 3]
```diff
@@ -180,9 +181,78 @@ const Context: GraphQLFieldConfig<
...
+      type: Artist.type,
+    },
+    basedOn: {
+      type: Artist.type,
+      resolve: ({ based_on }) => based_on,
+    },
+  }),
+})
+
+const HomePageArtworkOfFollowedArtistModuleType = new GraphQLObjectType<
```
Comment (alloy) [inline src/schema/home/context.ts]: @zephraph Perhaps this should just be `Artist` and we can add to the description of the `context` field that `Artist` represents a followed artist?

Comment (zephraph) [reply src/schema/home/context.ts]: Yeah, I'm not sure. The name is both very descriptive and a bit confusing so I'm definitely up for changing it. We could name the field `followedArtist` and just return the artist type?

File: src/schema/home/context.ts (hunk 2 above)
Comment (zephraph) [reply src/schema/home/context.ts]: I don't have any suggestions. I'm not in love with it, but it also seems good enough.

File: src/schema/home/context.ts (hunk 3 above)
Comment (zephraph) [reply src/schema/home/context.ts]: I don't know if my suggested made sense in the context of your question. I'd be wary of using a description on `context` as being the sole differentiator just because the description might be overlooked.

Comment (alloy) [reply src/schema/home/context.ts]: Fair enough, as well. I’ll leave this as-is then 👍

File: src/schema/home/context.ts (hunk 2 above)
Comment (alloy) [reply src/schema/home/context.ts]: Fair enough 👍

**Comment by alloy** [comment]
@zephraph Ok, this is ready for final review, but no merging yet please! I will do so myself after updating Emission to consume these changes.

**Comment by alloy** [comment]
Going to close this as we’re no longer going to be using transformers.
